app/bundles/EmailBundle/Assets/js/batch.js

```diff
@@ -0,0 +1,17 @@
+//EmailBundle (Copied from app/bundles/LeadBundle/Assets/js/lead.js)
+Mautic.emailBatchSubmit = function() {
+    if (Mautic.batchActionPrecheck()) {
+        if (mQuery('#email_batch_newCategory').val()) {
+            if (mQuery('#email_batch_ids').length) {
+                mQuery('#email_batch_ids').val(Mautic.getCheckedListIds(false, true));
+            }
+
+            return true;
+        }
+
+    }
+
+    mQuery('#MauticSharedModal').modal('hide');
+
+    return false;
```

Comment:
mmm, not sure. This part I copied from a similar screen

Reply:
I get that from the comment. Some old parts of Mautic aren't really up to the standard we want the new code to be in. It's always good to take a look if it can be improved after pasting. As some point someone will copy this code from you and it would be nice to have it in the best shape possible :)

Reply:
I'll keep that in mind for upcoming PRs ;)

Reply:
From what I can see, the return value and the hiding of the modal would be needed if there was no email selected. Something that shouldn't happen in the first place as the batch actions wouldn't have been shown.

Still, since the method `emailBatchSubmit` is invoked upon submitting the form to determine whether to submit or not, I don't think it's a bad idea to leave the `return false` in place.